src/styles/blocks/calendar.scss

```
@@ -0,0 +1,60 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+  max-width: 726px;
+
+  &__day {
+    display: flex;
+    width: 100px;
+    height: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
+
+    font-size: 30px;
+    font-family: Arial, sans-serif;
+    align-items: center;
+    justify-content: center;
+    transition-duration: 0.5s;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: #ffbfcb;
+    transform: translate(0, -20px);
+  }
+
+  $days: (
+    mon: 0,
+    tue: 101px,
+    wed: 202px,
+    thu: 303px,
+    fri: 404px,
+    sat: 505px,
+    sun: 606px,
```

The indentation must be calculated inside the loop.
```suggestion
    mon: 0,
    tue: 1,
    wed: 2,
    thu: 3,
    fri: 4,
    sat: 5,
    sun: 6,
```